IlmBase/IlmThread/IlmThreadMutexPosix.cpp

```diff
@@ -41,6 +41,10 @@
 
 #include "IlmBaseConfig.h"
 
+#if (defined(_WIN32) || defined(_WIN64))
```

Comment:
ditto in all similar

Reply:
See my above explanations for why I dropped them everywhere.